### PR TITLE
Move runpipe test to github actions

### PR DESCRIPTION
**Pull request by vmcj**
The logs can already be downloaded via the GitHub interface, so that part has been removed.

### CODE REVIEW

File: .github/jobs/chroot.bats [hunk 1]
```diff
@@ -9,6 +9,10 @@ fi
 # Cleanup old dir
 rm -rf $CHROOT
 
+if [ "${ARCH}" = 'default_placeholder' ]; then
```
Comment (meisterT) [inline .github/jobs/chroot.bats:12]: Please add a comment here why this is necessary.

File: .github/workflows/runpipe.yml [hunk 2]
```diff
@@ -0,0 +1,30 @@
+name: Run runpipe tests
+on:
+  push:
+    branches-ignore:
+      - main
```
Comment (meisterT) [inline .github/workflows/runpipe.yml:5]: Why do we ignore this on main and other branches?

Comment (vmcj) [reply .github/workflows/runpipe.yml:5]: We have the merge train, so we already had a PR where we ran this.

File: .github/jobs/chroot.bats (hunk 1 above)
Comment (vmcj) [inline .github/jobs/chroot.bats:12]: // In GHA CI we use this option to trigger all non specific architectures.

Comment (vmcj) [inline .github/jobs/chroot.bats:12]:
```suggestion
// In GHA CI we use this option to trigger all non specific architectures.
if [ "${ARCH}" = 'default_placeholder' ]; then
```

**Comment by meisterT** [comment]
LGTM once the failing test is fixed

**Comment by vmcj** [comment]
I took the failing part out as that seems to be interesting on itself.

**Pull request merged**